3-4/3-4.js

```diff
@@ -1,3 +1,4 @@
+// Guessing
```

Comment:
Remove these files from the `master` branch as all the codes in the `master` branch should be placed after being reviewed.

Reply:
I can't put comments after line 5 as it has already been placed in the `master` branch somehow. Let me put comments there after you delete them from the `master` branch.